include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -106,14 +89,6 @@ class fpga_policy : public device_policy<KernelName>
     fpga_policy(const fpga_policy<other_factor, OtherName>& other) : base(other.queue()){};
     explicit fpga_policy(sycl::queue q) : base(q) {}
     explicit fpga_policy(sycl::device d) : base(d) {}
-
-    // For internal use only
-
-    const base&
-    __device_policy() const
```

Comment:
@rarutyun Will we have here ABI breaking change?
We remove methods from public class...

Reply:
The answer is the same. If you have some doubts, let's discuss them offline.